vc_70.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+SAMPLE="AI-70_S61"
```

Comment:
At this moment you can copy the list of samples from Snake file and just iterate SAMPLE variable over it.
Moreover, in bash you can write:
```bash
for FILE in data/raw_reads/*.fastq.gz
```

And you will have all the analysis repeated for each file in the folder